src/setupWorker/start/utils/activateMocking.ts

```diff
@@ -7,7 +7,9 @@ export const activateMocking = async (
   context.worker?.postMessage('MOCK_ACTIVATE')
 
   return context.events.once('MOCKING_ENABLED').then(() => {
-    if (!options?.quiet) {
+    const isQuiet =
+      typeof options?.quiet === 'function' ? options?.quiet() : options?.quiet
```

Comment:
I don't think we should execute a `quiet` predicate function here. How about we only check the `options.quiet` for truthiness?

```js
// Basically, keeping the previous implementation.
if (!options?.quiet)
```

This is an internal log message not bound to any request. It should still abide by `quiet: true` in case you want to silence all logs, but it shouldn't support predicate as there is no request context to execute the predicate in. 

What are your thoughts on this?

Reply:
wouldn't the `options?.quiet` be a function?

Reply:
A good point, as a function value produces a truthy value, while the option describes a negative value (`true` for disable).

Reply:
This raises an entirely new concern: how can one have a custom logging predicate for requests and also disable the internal messages? The current `quiet` option value cannot handle this.

Reply:
either we make `req` and `res` in callback optional or introduce a new option